platform/kobo/fmon/README.txt

```diff
@@ -1,19 +1,23 @@
 Installation instructions for Kobo:
 
 
-Newer installation based on Start Menu:
-- Install "Start Menu" (http://www.mobileread.com/forums/showthread.php?t=255978)
-- Extract the koreader folder of the zip into the ".adds" directory (both KoboRoot.tgz and the koreader.png file are only needed for the older installation based on Filemonitor. See below.).
+Main installation method based on Start Menu:
```

Comment:
Oh, I normally never look in the fmon folder. I have been updating the install instructions here: https://github.com/koreader/koreader/wiki/Installation-on-Kobo-devices

(I prefer the "*Recommended* installation method based on Start Menu" phrasing)

Reply:
Yeah, given how outdated that file looked, I'm grateful there's something maintained elsewhere ;D.